.gitattributes

```diff
@@ -14,5 +14,5 @@
 *.swf  binary
 *.swc  binary
 
-# Fix GH saying Ruffle is written in ActionScript 
+# Fix GH saying Ruffle is written in ActionScript
```

Comment:
The comment, which predates us actually writing a lot of Ruffle in ActionScript, is a bit ironic now. Not sure if we should change it or what.

Reply:
Updated the comment